docs/api/environment-variables.md

```diff
@@ -53,16 +53,24 @@ Unsupported options are:
 
 ### `GOOGLE_API_KEY`
 
-You can provide an API key for making requests to Google's geocoding webservice. To do this, place the following code in your main process
-file, before opening any browser windows that will make geocoding requests:
+You can provide an API key for making requests to Google Cloud Platform's
+[geolocation webservice](https://developers.google.com/maps/documentation/geolocation/intro).
+To do this, place the following code in your main process file, before opening any browser
+windows that will make geolocation requests:
 
 ```javascript
 process.env.GOOGLE_API_KEY = 'YOUR_KEY_HERE'
 ```
 
-For instructions on how to acquire a Google API key, visit [this page](https://developers.google.com/maps/documentation/javascript/get-api-key).
-By default, a newly generated Google API key may not be allowed to make
-geocoding requests. To enable geocoding requests, visit [this page](https://developers.google.com/maps/documentation/geocoding/get-api-key).
+For instructions on how to acquire a Google API key, visit
+[this page](https://developers.google.com/maps/documentation/geolocation/get-api-key).
+By default, a newly generated Google API key may not be allowed to make geolocation requests.
+To enable the geolocation webservice for your project, enable it through the
+[API library](https://console.cloud.google.com/apis/library).
+
+N.B. You will need to add a Billing Account to your project before being able to use your
```

Comment:
I'm generally not a fan of links that say "here" or "this page". This is my suggestion to avoid it:

```suggestion
N.B. You will need to add a [Billing Account](https://cloud.google.com/billing/docs/how-to/payment-methods#add_a_payment_method) to your project before being able to use your
```